DotNetExcel/Resources/corporateAttachment.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace StarkBankExcel
+{
+    internal class corporateAttachment
+    {
+        public static JObject Get(string key)
+        {
+
+            return V2Request.Fetch(
+                V2Request.Get,
+                Globals.Credentials.Range["B3"].Value,
+                "attachment/" + key + "?expand=content"
```

Comment:
faz seguindo o Invoice.Get
`public static JObject Get(string cursor = null, Dictionary<string, object> optionalParams = null)
        {
            string query = "";

            if (cursor != null)
            {
                query = "?cursor=" + cursor;
            }

            if (optionalParams != null)
            {
                foreach (string key in optionalParams.Keys)
                {
                    if (query == "")
                    {
                        query = "?" + key + "=" + optionalParams[key].ToString();
                    }
                    else
                    {
                        query = query + "&" + key + "=" + optionalParams[key].ToString();
                    }
                }
            }

            return V2Request.Fetch(
                V2Request.Get,
                Globals.Credentials.Range["B3"].Value,
                "invoice/" + query
            ).ToJson();
        }`

Reply:
eu ia fzr assim, porem esse requests de attachment tem um padrão diferente de request